requirements.txt

```diff
@@ -1,10 +1,10 @@
 argcomplete==1.12.2
 colorama==0.4.4
-flake8==3.8.4
+flake8==4.0.1
 jmespath==0.10.0
 Pygments==2.8.1
-pylint==2.7.2
-pytest==6.2.2
+pylint==2.11.1
+pytest==6.2.5
```

Comment:
Old `pytest` fails in Python 3.10: https://github.com/pytest-dev/pytest/issues/8546#issuecomment-820232967

```
=============================================== short test summary info ===============================================
ERROR tests/test_cli_scenarios.py - TypeError: required field "lineno" missing from alias
ERROR tests/test_command_registration.py - TypeError: required field "lineno" missing from alias
ERROR tests/test_command_with_configured_defaults.py - TypeError: required field "lineno" missing from alias
ERROR tests/test_completion.py - TypeError: required field "lineno" missing from alias
ERROR tests/test_config.py - TypeError: required field "lineno" missing from alias
ERROR tests/test_deprecation.py - TypeError: required field "lineno" missing from alias
ERROR tests/test_experimental.py - TypeError: required field "lineno" missing from alias
ERROR tests/test_help.py - TypeError: required field "lineno" missing from alias
ERROR tests/test_introspection.py - TypeError: required field "lineno" missing from alias
ERROR tests/test_log.py - TypeError: required field "lineno" missing from alias
ERROR tests/test_output.py - TypeError: required field "lineno" missing from alias
ERROR tests/test_parser.py - TypeError: required field "lineno" missing from alias
ERROR tests/test_preview.py - TypeError: required field "lineno" missing from alias
ERROR tests/test_prompting.py - TypeError: required field "lineno" missing from alias
ERROR tests/test_query.py - TypeError: required field "lineno" missing from alias
ERROR tests/test_util.py - TypeError: required field "lineno" missing from alias
```

However, new `pytest` is more strict on grammar. We need to fix these additional check failures.